.github/workflows/main.yml

```diff
@@ -0,0 +1,11 @@
+name: A workflow for my Hello World file
+on: push
+jobs:
+  build:
+    name: Hello world action
+    runs-on: ubuntu-latest
```

Comment:
Notice you are being delivered a Github hosted ubuntu virtual machine equipped with most things you need to build code for many languages and interact with github.  The most convenient `runs-on` choice.